CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Eclipse LSP4Jakarta
+
+We welcome contributions, and request you follow these guidelines.
+
+ - [Raising issues](#raising-issues)
+ - [Legal](#legal)
+ - [Coding Standards](#coding-standards)
+ - [Developing](#developing))
+
+
+## Raising issues
+
+Please raise any bug reports on the [issue tracker](https://github.com/eclipse/lsp4jakarta/issues). Be sure to search the list to see if your issue has already been raised.
+
+A good bug report makes it easy for us to understand what you were trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
+
+## Legal
+
+In order to make contribution as easy as possible, we follow the [Developer's Certificate of Origin 1.1 (DCO)](https://developercertificate.org/) process for code contributions. We ask you to certify your contributions by signing off in your pull requests.
```

Comment:
This needs to be updated.  We follow the Eclipse process for contributors: https://www.eclipse.org/projects/handbook/#contributing-contributors.

Reply:
Updated the entire contributing document to use the generated document from the Eclipse Foundation.